tables/customizing-the-table-toolbar.html

```diff
@@ -109,10 +107,10 @@
                     menuActionManager.insertRowAfter();
                 });
 
-                // TinyMCE 5.4 introduced improved APIs for inserting tables.
+                // TinyMCE has APIs for inserting tables.
                 // Here's a toolbar button that displays a table grid picker
                 // The table picker is unfortunately undocumented at this stage
```

Comment:
Feels weird to just say this. Is this something we _want_ to document? Looks like we don't document fancy menu items at all. "At this stage" implies that we might document it down the track...

Reply:
Alternatively - why do we have an "inserttable" menu item but not a toolbar button in TinyMCE? Should we add one?? Questions for outside the scope of this PR.

Reply:
It's a question that comes up occasionally from customers as well. I sometimes regret my inability to put us on a good architecture with TinyMCE 5 that would have made both work automatically.